FABulous/FABulous_CLI/FABulous_CLI.py

```diff
@@ -1072,13 +1144,14 @@ def do_gen_user_design_wrapper(self, args: argparse.Namespace) -> None:
     @with_category(CMD_TOOLS)
     @with_argparser(gen_tile_parser)
     def do_generate_custom_tile_config(self, args: argparse.Namespace) -> None:
-        """Generates a custom tile configuration for a given tile folder or path to bel
-        folder. A tile .csv file and a switch matrix .list file will be generated.
+        """Generate a custom tile configuration for a given tile folder.
+
+        Or path to bel folder. A tile .csv file and a switch matrix .list file will be
```

Comment:
Did this slip through in my last review?
```suggestion
        Or path to BEL folder. A tile '.csv' file and a switch matrix '.list' file will be
```